scania-digital-design-system/sdds
# feat(tabs): refactor tabs

**Describe pull-request**  
Refactoring the tabs. We now have: inline-tabs, folder-tabs and navigational-tabs, they all have two subcomponents (button and link). This PR also includes removing the functionality of displaying the tabs content. The fallback tab label was also removed in this PR. 

**Solving issue**  
Fixes: #3221 #3173

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Tabs -> Folder Tabs / Inline Tabs / Navigational Tabs
3. Check that the styling is correct. 

**Suggested test steps**

**Screenshots**  
-

**Additional context**  
-
